pages/common/tmux.md

```diff
@@ -19,17 +23,13 @@
 
 `tmux a`
 
-- Attach to a named session:
-
-`tmux a -t {{name}}`
-
 - Detach from session:
 
 `Ctrl + B, D`
 
 - Kill session:
 
-`tmux kill-session -t {{name}}`
+`tmux kill-session`
```

Comment:
hi @robbyrenz , how about change the command description to `Kill the given session` while not change the command?

Reply:
Okay, sure, I'll make the changes, thanks for the feedback.